crates/bevy_render/src/render_graph/base.rs

```diff
@@ -213,29 +384,42 @@ pub(crate) fn add_base_graph(config: &BaseRenderGraphConfig, world: &mut World)
                 TextureDescriptor {
                     size: Extent3d {
                         depth: 1,
-                        width: 1,
-                        height: 1,
+                        width: 2560,
+                        height: 1440,
```

Comment:
Why the hardcoded width/height here?

Reply:
Missed that one during refactoring to use the newer WindowTextureNodes.